ci-runtests.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+set -eu
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$SCRIPT_DIR"
+
+git pull
```

Comment:
I'm not sure if updating the repo like this is sensible. Even if it is, it wouldn't hurt to verify that the last commit is signed before merging it into the current branch.

Reply:
I don't know of a better way to do this.